study-builder/studies/shared/family-history/snippets/family-history-intro-graphic.conf

```diff
@@ -0,0 +1,11 @@
+{
+  "titleTemplate": null,
+  "bodyTemplate": {
+    "templateType": "HTML",
+    "templateText": """<div class="top-graphic"></div><div class="second-graphic"></div>""",
```

Comment:
Breaking out the graphics like this _might_ not be ideal for mobile client. I _think_ mobile client display each block as a screen, so this might result in a blank screen since mobile client doesn't really consume `div`s / `class`es? Maybe we should just merge this one into `family-history-section-intro.conf`?

Reply:
@yufengwng is right, this will result in a blank screen on the mobile client. Why do we need this block? There's no content in it- it's purely presentation.

Reply:
Right These html`div`s are purely presentation. The design calls for top of this page (the introduction section) to look like below.
On the Angular side we were planning to have CSS images that live on Angular client to be used for these classes.

Would not be a big deal to merge the divs into same CONTENT block as the rest of the introduction. Would this refactoring take care of problem?

<img width="1159" alt="Screen Shot 2021-03-05 at 3 55 23 PM" src="https://user-images.githubusercontent.com/29984380/110172839-b7c66e00-7dcb-11eb-8ed7-072ec5631091.png">

Reply:
I believe it'll be better to merge them into one CONTENT block.

Reply:
Potentially not- Will there be any effect of a client does not have the CSS stylesheets for those classes? The native apps _are_ expecting HTML content in the `CONTENT` blocks. If those are effectively NOPs with the classes missing, we should be alright.

Reply:
I'll merge the blocks then.
@bskinner I went with divs with class names to leave it up to client to show whatever it wants. On Angular we are obviously planning on having CSS to include Erin's graphics.
The default styling for divs as I understand on Chrome and Firefox is to display nothing, but can't say I know what to expect from native mobile clients. Open to suggestions and requests.

Reply:
There should be similar behavior, so we should be good with a merge. Android uses a fork of the Blink rendering engine (same as Chromium), and everything on iOS uses Apple's WebKit.

Reply:
Moved graphics into intro block 435fad480cf241d07557723b770904cc3a095bd5